### ISSUE TITLE
Question (not a problem): is /84 command supposed to return short status output ?

### ISSUE THREAD

**Issue by peoyli** [nielsonm236/NetMod-ServerApp#165]
I noticed that the /84- and /84+ commands also returns the current short form status. Is that intentional ?

If status output is left in, I can skip the /99 command in my init code for the IO module, and just use /84- to get the status the same time I set the format to show disabled ports as '-'.

Sample init code (part of my project) in PHP:

```
<?php
  error_reporting(E_ALL ^ E_WARNING);
  $module_url = "http://192.168.1.4:8080";
  $default_socket_timeout = ini_get('default_socket_timeout');
  ini_set('default_socket_timeout', 2);
  // Turn all ports off and keep track of the time
  $time_start = microtime(true);
  $r = file_get_contents("{$module_url}/56");
  $time_end = microtime(true);

  if ($time_end - $time_start > 1)
    die("Timeout, module is DOWN\n");

  // Get status
  $r = file_get_contents("{$module_url}/84-");
  print "$r\n";
  ini_set('default_socket_timeout', $default_socket_timeout);
?>

```

**Comment by nielsonm236** [comment]
It is intentional that /84 returns the short form so that you can see the result immediately.

**Comment by peoyli** [comment]
Fully acceptable. If someone doesn't want the result, just ignore it.
Should be added to the documentation for clarity.

**Comment by nielsonm236** [comment]
Addressed in release 20230416 1116
